http/http.go

```diff
@@ -34,6 +34,8 @@ type Server struct {
 type Response struct {
 	IP         net.IP               `json:"ip"`
 	IPDecimal  *big.Int             `json:"ip_decimal"`
+	RegionName string               `json:"region_name,omitempty"`
+	RegionCode string               `json:"region_code,omitempty"`
```

Comment:
Can you move these fields just before PostalCode below?

Reply:
I think thematically those fields and the postalcode would fit between Country and City. I would put postalcode last as some cities have multiple.